### PR TITLE
Pagination fixes

### PR DESCRIPTION
**Pull request by muhammad-ammar**
* Updated docstring to mention user anonymous id
* Added `start` param in paginator. This is needed for Backbone paginator to correctly show the header message.
* Replaced `current` param in paginator with `current_page`

### CODE REVIEW

**Comment by muhammad-ammar** [comment]
@symbolist @muzaffaryousaf PTAL. Thanks!

**Comment by muzaffaryousaf** [comment]
LGTM :+1: Please add description to this PR.

File: notesapi/v1/paginators.py [hunk 1]
```diff
@@ -18,7 +18,8 @@ def get_paginated_response(self, data):
         Annotate the response with pagination information.
         """
         return Response({
-            'current': self.page.number,
+            'start': (self.page.number - 1) * self.get_page_size(self.request),
```
Comment (symbolist) [inline notesapi/v1/paginators.py]: Is this supposed to be zero indexed?

Comment (muhammad-ammar) [inline notesapi/v1/paginators.py]: yes, this is what we are using in LMS.

**Comment by symbolist** [comment]
Looks good then. :+1:

**Pull request merged**